build.cake

```diff
@@ -1,6 +1,7 @@
-#addin "Cake.Git&version=1.1.0"
-#tool "dotnet:?package=GitVersion.Tool&version=5.8.1"
-#tool "nuget:?package=xunit.runner.console&version=2.4.1"
+#addin "Cake.Git&version=2.0.0"
+#tool "dotnet:?package=GitVersion.Tool&version=5.10.3"
+#tool "nuget:?package=xunit.runner.console&version=2.4.2"
+#tool "nuget:?package=NuGet.CommandLine&version=6.3.1"
```

Comment:
Yes, otherwise the `NugetPush` command failes due to missing nuget.exe
```
An error occurred when executing task 'Publish'.
Error: Could not locate nuget.exe.
```